modules/bim/app/workers/bim/ifc_models/ifc_conversion_job.rb

```diff
@@ -6,6 +6,8 @@ class IfcConversionJob < ::ApplicationJob
       ##
       # Run the conversion of IFC to
       def perform(ifc_model)
+        return retry_job(wait: 1.minute) if attachment_missing?(ifc_model)
```

Comment:
Will this retry indefinitely or give up at some point when the direct upload actually failed for some reason?